src/doomtype.h

```diff
@@ -90,6 +91,18 @@ typedef byte lighttable_t;
 #define inline __inline
 #endif
 
+#if defined(__clang__)
+ #ifndef __has_feature
+  #define __has_feature(x) 0
```

Comment:
Why do we even need this? This is a clang-only feature and the clang docs are pretty clear about the check.

Reply:
I'v removed it now.